src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy

```diff
@@ -145,14 +182,26 @@ public class AbstractFeatureTask extends AbstractServerTask {
         }
 
         Set<String> featuresToInstall = InstallFeatureUtil.combineToSet(pluginListedFeatures, dependencyFeatures, serverFeatures)
-        return featuresToInstall 
+        return featuresToInstall
+    }
+
+    private void createNewServerFeatureUtil() {
+        servUtil = new ServerFeatureTaskUtil();
+        return;
```

Comment:
The `return` statement isn't strictly needed here.  Then since this is one line, might as well just put it directly in line 195.